src/clr-angular/data/datagrid/built-in/filters/datagrid-numeric-filter.ts

```diff
@@ -165,7 +157,7 @@ export class DatagridNumericFilter<T = any> extends DatagridFilterRegistrar<T, D
 
   @Output('clrFilterValueChange') filterValueChange = new EventEmitter();
 
-  public close() {
-    this.open = false;
-  }
+  // public close() {
+  //   this.open = false;
+  // }
```

Comment:
should we remove this if not being used? or is it temporarily commented out?

Reply:
yes, it should be removed.

Reply:
This has been removed in the feedback commit.